Single header include failure

With the latest release I'm getting:
```cpp
/Users/tdegeus/miniconda3/envs/test/include/xtensor/zarray.hpp:19:10: fatal error: 'zassign.hpp' file not found
#include "zassign.hpp"
         ^~~~~~~~~~~~~
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -182,6 +182,7 @@ set(XTENSOR_HEADERS
     ${XTENSOR_INCLUDE_DIR}/xtensor/xview_utils.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/zarray.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/zarray_impl.hpp
+    ${XTENSOR_INCLUDE_DIR}/xtensor/zassign.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/zdispatcher.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/zdispatching_types.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/zfunction.hpp

